types/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Minimum TypeScript Version: 3.2
+import {Node} from 'unist'
+
+declare namespace toHast {
+  type Handler = (h: any, node: Node) => any
```

Comment:
I wasn’t sure about the type of `h`. I’d prefer to see this merged and released fast with an `any` type, so I can use this to add types to `remark-rehype`, which is what I’m really interested in.

Reply:
`h` is defined here https://github.com/syntax-tree/mdast-util-to-hast/blob/48d420d10cd142a6266d4dee3eb76d804b5fe623/lib/index.js#L33-L39